x-pack/plugins/cases/public/components/all_cases/selector_modal/use_cases_add_to_existing_case_modal.test.tsx

```diff
@@ -89,4 +125,47 @@ describe('use cases add to existing case modal hook', () => {
       })
     );
   });
+
+  it('should call postComment when a case is selected', async () => {
+    const mocked = jest.fn();
+    usePostCommentMock.mockReturnValueOnce({
+      postComment: mocked,
+    });
+
+    AllCasesSelectorModalMock.mockImplementation(({ onRowClick }) => {
+      onRowClick({ id: 'test' } as Case);
+      return null;
+    });
+
+    const result = appMockRender.render(<TestComponent />);
+    userEvent.click(result.getByTestId('open-modal'));
+
+    await waitFor(() => {
+      expect(mocked).toHaveBeenCalledWith({
+        caseId: 'test',
+        data: alertComment,
+        throwOnError: true,
+      });
+    });
+  });
+
+  it('should not call postComment when a case is selected but no attachments are passed', async () => {
```

Comment:
Could you please add a test that tests that the toast has been called on an error (this is basically the bug we trying to fix)?

Reply:
I'll add a test to check that the success toast had not been called. The error toast is responsability of the post comment hook.

Reply:
Yes sorry that's what I meant 🙂 . My mistake.

Reply:
Nevertheles I'll add the test for the error toast in the postmessage hook. It is not tested there.

Reply:
Added